docs/metrics.md

```diff
@@ -205,6 +205,9 @@ curl http://localhost:8000/metrics
 ```
 [Example metrics output](https://raw.githubusercontent.com/openvinotoolkit/model_server/v2022.2/docs/metrics_output.out)
 
+## Performance considerations
+Collecting metrics has negligible performance overhead when used with typical models. However when used with very lightweight, fast models which inference time is very short, the metric incrementation can take noticeable proportion of the processing time. Consider it while enabling metrics for such models.
```

Comment:
Maybe instead of "typical models" we could be more precise? Like replace it with something like:
"models of average size and complexity". It would fit nicely with the next sentence when you mention "very lightweight, fast models"

Reply:
fixed